mobile/src/main/res/values/strings.xml

```diff
@@ -236,7 +236,10 @@
     <string name="settings_dnd_mode_summary_on">Sends DND mode to Item \'%1$s\'</string>
     <string name="settings_dnd_mode_summary_off">Don\'t send DND mode</string>
     <string name="settings_dnd_mode_howto_url" translatable="false">https://www.openhab.org/docs/apps/android.html#dnd-mode</string>
+    <string name="settings_background_tasks_permission_hint">You\'ll be asked for a required permissions after turning on this setting. Please do not deny the permissions.</string>
+    <string name="settings_background_tasks_permission_hint_android_r">You\'ll be asked for a required permissions after turning on this setting. Please allow it for all the time.</string>
```

Comment:
Wording improvement suggestion: `This feature requires additional permissions to work. After enabling it, you'll be asked for permission approval.` ... and then for pre-R remove the second sentence, and for post-R `Please make sure to approve the permission also for background usage, since the Item updates also happen when the app is in background.`

Reply:
I also changed the string names to be consistent with other strings that have different versions on different api levels.